neigh_linux.go

```diff
@@ -42,11 +42,12 @@ const (
 
 // Neighbor Flags
 const (
-	NTF_USE    = 0x01
-	NTF_SELF   = 0x02
-	NTF_MASTER = 0x04
-	NTF_PROXY  = 0x08
-	NTF_ROUTER = 0x80
+	NTF_USE         = 0x01
+	NTF_SELF        = 0x02
+	NTF_MASTER      = 0x04
+	NTF_PROXY       = 0x08
+	NFT_EXT_LEARNED = 0x10
```

Comment:
looks like you have a typo here

https://elixir.bootlin.com/linux/v5.11/source/include/uapi/linux/neighbour.h#L47